### ISSUE TITLE
Modifications not loading from a definition file of a loaded folder.

### ISSUE THREAD

**Issue by WillyB1337** [RigsOfRods/rigs-of-rods#2674]

**Comment by tritonas00** [comment]
same as https://github.com/RigsOfRods/rigs-of-rods/issues/2452 ?

**Comment by WillyB1337** [comment]
**### _OOPS_**

**Comment by WillyB1337** [comment]
https://github.com/RigsOfRods/rigs-of-rods/issues/2675#issue-803203320
Duplicate of #

**Comment by WillyB1337** [comment]
So apparently, as of the latest builds (Can't tell which one), the feature is not working anymore.

Is there any workaround?

This is my current build : **Rigs of Rods 2021.02-dev-3b58406 (RoRnet_2.42) [2021-02-07, 19:13]**
Its the old DevEnv release.

**Comment by only-a-ptr** [comment]
The directories/ZIPs under ROR\mods are tracked by ModCache and loaded only once; reloading from there has always been a mess.

Instead of fixing this I'll make RoR treat all mods read-only and finish my "project directories" concept: https://github.com/RigsOfRods/rigs-of-rods/pull/2677
